samples/excel/46-table/sort-data.yaml

```diff
@@ -13,56 +13,55 @@ script:
 
         async function sortTable() {
             await Excel.run(async (context) => {
-            const sheet = context.workbook.worksheets.getItem("Sample");
-            const expensesTable = sheet.tables.getItem("ExpensesTable");
-
-            // sort the table by the "Amount" column
-            const sortFields = [
-                {
-                key: 3,
-                ascending: false
-                }
-            ];
-            expensesTable.sort.apply(sortFields);
-
-            await context.sync();
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const expensesTable = sheet.tables.getItem("ExpensesTable");
```

Comment:
Were these just always badly indented and you're now fixing them?

Reply:
Yeah, figured I might as well clean this up while I'm in there.